runtime/compiler/runtime/J9Runtime.hpp

```diff
@@ -169,6 +169,12 @@ typedef struct TR_AOTMethodHeader {
    uintptr_t compileMethodDataSize;
    uintptr_t unused;
```

Comment:
Do you why the `unused` exists? can be eliminated?
Also, for better packing and smaller headers we should probably move the flags after the last `uint32_t` entry.
Also, I think we need to bump TR_AOTMethodHeader_MajorVersion since its structure has changed.

Reply:
> Do you why the unused exists? can be eliminated?

Looks like there's a PR out to remove it https://github.com/eclipse/openj9/pull/11736

> Also, for better packing and smaller headers we should probably move the flags after the last uint32_t entry.
> Also, I think we need to bump TR_AOTMethodHeader_MajorVersion since its structure has changed.

Ok will do.